processing/dwf_calculation_algorithm.py

```diff
@@ -178,6 +178,7 @@ def dwf_json_to_csv(dwf_list, output_csv_file):
 
     with open(output_csv_file, "w", newline="") as csv_file:
         writer = csv.writer(csv_file)
+        writer.writerow(["id","connection_node_id","timeseries"])
```

Comment:
@leendertvanwolfswinkel Please add spaces after comas.
```suggestion
        writer.writerow(["id", "connection_node_id", "timeseries"])
```